go/vt/sqlparser/ast.go

```diff
@@ -3813,11 +3815,24 @@ func (node *Flush) Format(buf *TrackedBuffer) {
 		buf.Myprintf(" %s", strings.ToLower(node.Type))
 	}
 
-	if node.Option.Name == "RELAY LOGS" && node.Option.Channel != "" {
+	if strings.ToLower(node.Option.Name) == "relay logs" && node.Option.Channel != "" {
```

Comment:
can use `strings.EqualsFold()` instead

Reply:
oooh... that's a new util for me. thanks for sharing! I'll switch over to that.